### DISCUSSION TITLE
restore after delete all calendar date possible?

### DISCUSSION THREAD

**Discussion by delay77** [Etar-Group/Etar-Calendar#1072] [General]
Hello,
thank you for the friendly welcome.
perhaps this Diskussion i wrote at a wrong place. Sorry if it is, and sorry for my poor english.
because of a mistake, all my calendardates will be delete, not the calendar himself.
Unfortunatly there was no Synchronisation with Google.
Calendar is used in lineageos Android 10.
Under the emailadress From calender in settings calendarinformation there where 400 dates listet, but only one is still in the overview.
Is it possible to reconstruct all dates or are all data lost and overwrite?
Thank you very mutch for answers and help.
Thomas

**Comment by jspricke** [comment]
Hi,

Etar does not store calendar data itself but relies on the Android calendar provider: https://github.com/Etar-Group/Etar-Calendar#technical-explanation
I'm not aware of any internal backup for that.
